src/models/metadata.js

```diff
@@ -0,0 +1,30 @@
+import { Schema, model } from "mongoose";
+
+const bookReviewsAndRatingSchema = new Schema({
+    reviewId: {
```

Comment:
We can just name this as `id`

Reply:
yes initially i thought same, but tried this way.